lib/index.ts

```diff
@@ -174,6 +266,14 @@ export class Front {
 
 		return newPath;
 	}
+
+	private validateEventSignature(data: any, signature: string): boolean {
+		const hash = crypto.createHmac('sha1', this.apiSecret)
+			.update(JSON.stringify(data))
```

Comment:
For some reason I'd got it into my head this was called from a Promise (which obviously it isn't!). Yep, I'll wrap it in a `try`.